### PR TITLE
System created

### PR DESCRIPTION
**Pull request by BrunoSoaresGH**

### CODE REVIEW

File: src/components/CardList/CardList.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    list-style-type: none;
+    padding-left: 24px;
+    padding-top: 24px;
+  }
+  .card-list-item {
+    margin: 12px;
+    max-width: 240px;
+    max-height: 400px;
+    display: inline;
+  }
```
Comment (edimossilva) [inline src/components/CardList/CardList.css]: add an empty line to the end of all your files

File: src/components/Card/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import "./Card.css";
+
+class Card extends Component{
+    render(){
```
Comment (edimossilva) [inline src/components/Card/Card.jsx:5]: is your lint working properly? in some files I see 2 spaces in other files I see 4 spaces

File: src/components/CardList/CardList.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import Card from "../Card/Card";
+import "./CardList.css";
+
+class CardList extends Component{
```
Comment (edimossilva) [inline src/components/CardList/CardList.jsx:5]: No, your lint is not working properly

File: src/components/CardList/CardList.css (hunk 1 above)
Comment (marceloglacial) [reply src/components/CardList/CardList.css]: There's a option on vs code to do it automatically

File: src/components/Card/Card.jsx (hunk 2 above)
Comment (edimossilva) [reply src/components/Card/Card.jsx:5]: I've created an issue #3  related to this, let take care of this in another PR

File: src/components/CardList/CardList.css (hunk 1 above)
Comment (BrunoSoaresGH) [reply src/components/CardList/CardList.css]: done

**Review by lkzfox — APPROVED** [review]

**Review by edimossilva — APPROVED** [review]

**Pull request merged**